.github/workflows/cd.yml

```diff
@@ -114,7 +117,7 @@ jobs:
         with:
           python-version: 3.8
       - name: Create distribution
-        run: python setup.py sdist
+        run: python setup.py sdist bdist_wheel
```

Comment:
Why do we need a binary distribution?

Reply:
https://pythonwheels.com/ ok i read a bit here

Reply:
Yeah and I suspect this is why it's not possible to run
```ipython
!pip install expertsystem==0.6.2
```
on Google Colab